ci-operator/step-registry/upi/deprovision/heterogeneous/powervs/upi-deprovision-heterogeneous-powervs-commands.sh

```diff
@@ -1,37 +1,50 @@
 #!/bin/bash
 
-# Generates a workspace name like rdr-multi-arch-4-14-au-syd-n1
-# this keeps the workspace unique per version
-OCP_VERSION=$(cat "${SHARED_DIR}/OCP_VERSION")
-REGION="${LEASED_RESOURCE}"
-CLEAN_VERSION=$(echo "${OCP_VERSION}" | tr '.' '-')
-if [ ! -z "${WORKFLOW_TYPE}" ]
+##### Constants
+IBMCLOUD_HOME=/tmp/ibmcloud
+export IBMCLOUD_HOME
+
+IBMCLOUD_HOME_FOLDER=/tmp/ibmcloud
+export IBMCLOUD_HOME_FOLDER
+
+export PATH=$PATH:/tmp:"${IBMCLOUD_HOME_FOLDER}"
+
+# Variables
+# OCP Version
+if [ ! -f "${SHARED_DIR}/OCP_VERSION" ]
 then
-  CUCUSHIFT="-cucushift"
-else
-  CUCUSHIFT=""
+    echo "Provisioning never happened"
+    exit 0
```

Comment:
If provisioning never happens, wouldn't we want to report failure, not success, from this step?

Reply:
it's deprovision